package_control/ca_certs.py

```diff
@@ -42,19 +41,19 @@ def get_ca_bundle_path(settings):
         regenerate = regenerate or os.path.getmtime(user_ca_bundle_path) > os.path.getmtime(merged_ca_bundle_path)
 
     if regenerate:
-        with open(merged_ca_bundle_path, 'wb') as merged:
+        with open(merged_ca_bundle_path, 'w', encoding='utf-8') as merged:
             if system_ca_bundle_path:
-                with open_compat(system_ca_bundle_path, 'r') as system:
-                    system_certs = read_compat(system).strip()
-                    merged.write(system_certs.encode('utf-8'))
+                with open(system_ca_bundle_path, 'r', encoding='utf-8', errors='replace') as system:
```

Comment:
I have no clue about the ca bundles code, but do we really want `errors='replace'` here and below?

Reply:
That's what `open_compat` used to do and I didn't intend to change behavior.

I can barely remember of an issue with regards to certain unicode characters contained in certificates but I am not aware of whether using `replace` was a fix for it. Basically I wonder, whether those could simply be copied in binary mode, but I guess, it would make stripping harder.

Reply:
This behavior of `open_compat()` exists since day 1 (9432d8184a1b64c4b541ca1bd18ffda9e63e2875) of ST3 support.

Reply:
We can kill `errors='replace'`. I'd rather get an exception and know about it.